bundles/pubsub/pubsub_admin_tcp/src/pubsub_psa_tcp_constants.h

```diff
@@ -102,21 +103,30 @@
  */
 #define PUBSUB_TCP_STATIC_CONNECT_URLS          "tcp.static.connect.urls"
 
+
 /**
- * Name of environment variable with space-separated list of ips/urls to connect to
- * e.g. PSA_TCP_STATIC_CONNECT_FOR_topic_scope="tcp://127.0.0.1:4444 tcp://127.0.0.2:4444"
+ * Defines if the publisher / subscriber is a passive endpoint and shares
```

Comment:
Can this be explained more. What is meant by passive. I am assuming not taking part in discovery.. correct?

And what are the possible values for tcp.passive.configured / tcp.passive.key

Reply:
I have added more comments to explain the concept.
It's the same concept that is used in the web socket.
See also the tcp endpoint test